test/e2e/simple_test.go

```diff
@@ -26,19 +27,25 @@ func TestE2ESimple(t *testing.T) {
 		t.Skip("skipping e2e test")
 	}
 
-	if os.Getenv("E2E_VERSIONS") != "" {
-		versionsStr := os.Getenv("E2E_VERSIONS")
-		versions := ParseVersions(versionsStr)
-		if len(versions) > 0 {
-			latestVersion = versions.GetLatest().String()
+	if os.Getenv("E2E_LATEST_VERSION") != "" {
```

Comment:
[Question] Isn't it needed to introduce the usage of the `E2E_LATEST_VERSION` env variable in the e2e readme?